samples/BankAccount/BankClient/Program.cs

```diff
@@ -39,19 +38,25 @@
         var fromBalance = await from.GetBalance();
         var toBalance = await to.GetBalance();
 
-        Console.WriteLine($"We transfered 100 credits from {fromName} to {toName}.\n{fromName} balance: {fromBalance}\n{toName} balance: {toBalance}\n");
+        Console.WriteLine(
+            $"We transfered 100 credits from {fromName} to " +
+            $"{toName}.\n{fromName} balance: {fromBalance}\n{toName} balance: {toBalance}\n");
     }
     catch (Exception exception)
     {
-        Console.WriteLine($"Error transfering 100 credits from {fromName} to {toName}: {exception.Message}");
+        Console.WriteLine(
+            $"Error transfering 100 credits from "+
+            $"{fromName} to {toName}: {exception.Message}");
+
         if (exception.InnerException is { } inner)
         {
-            Console.WriteLine($"\tInnerException: {inner.Message}\n");
+            Console.WriteLine(
```

Comment:
nit: can this be combined to one line? It looks like there is a longer line on https://github.com/dotnet/orleans/pull/7543/files#diff-8177308233ff4a774788a6d057e3815a9a01d7d3cc41e3ef6dc82e51a6d4727eR43

and this break IMO doesn't improve readability